R/HclustParam.R

```diff
@@ -91,12 +95,19 @@ setMethod("[[", "HclustParam", function(x, i) {
 setMethod("show", "HclustParam", function(object) {
     object <- updateObject(object)
     callNextMethod()
-    fun <- object@dist.fun
-    if (!is.null(fun)) {
+    dist.fun <- object@dist.fun
+    if (!is.null(dist.fun)) {
         cat("dist.fun: custom\n")
     } else {
         cat("dist.fun: stats::dist\n")
```

Comment:
In my understanding it is the general recommendation to use "message" instead of "cat" within functions. This related at least to ability to choose using some message suppression functions more easily when the user chooses so.

Reply:
If this question concerns bluster pkg more broadly then we would need the LTLA comment and possible another issue & PR for that (if any).

Reply:
Is message() used also when the object is printed? (im not sure)

Reply:
I believe LTLA only uses `cat` in his `show` functions. Perhaps this could be changed but indeed we would have to check with him.

Reply:
Ok that's another topic for later then.